tools/labs/Makefile

```diff
@@ -31,6 +33,24 @@ copy: $(YOCTO_IMAGE)
 	sudo umount $(TEMPDIR)
 	rmdir $(TEMPDIR)
 
+scp: ip
+	scp -r skels root@$(shell cat .ip):
```

Comment:
`scp` should only copy the `.ko` files, like the `make copy` command.

Reply:
`scp` can't create parent directories and we don't have `rsync` on the provided images. I can do it by `ssh`ing first to create the required directories and then using `scp` to copy the files. Would that be ok?

Reply:
That sounds ok.